include/zephyr/bluetooth/l2cap.h

```diff
@@ -453,14 +455,15 @@ struct bt_l2cap_server {
 	 *  authorization.
 	 *
 	 *  @param conn The connection that is requesting authorization
+	 *  @param server Pointer to the server structure this callback relates to
 	 *  @param chan Pointer to received the allocated channel
 	 *
 	 *  @return 0 in case of success or negative value in case of error.
 	 *  @return -ENOMEM if no available space for new channel.
 	 *  @return -EACCES if application did not authorize the connection.
 	 *  @return -EPERM if encryption key size is too short.
 	 */
-	int (*accept)(struct bt_conn *conn, struct bt_l2cap_chan **chan);
+	int (*accept)(struct bt_conn *conn, struct bt_l2cap_server* server, struct bt_l2cap_chan **chan);
```

Comment:
This will be a stable API change, and will need to follow the steps defined by https://docs.zephyrproject.org/latest/develop/api/api_lifecycle.html#stable